core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -505,7 +525,7 @@ public String buildKeyName(long keynum) {
     if (!orderedinserts) {
       keynum = Utils.hash(keynum);
     }
-    return "user" + keynum;
+	return "user"+String.format("%0"+zeropadding+"d",keynum);
```

Comment:
Just to note that format is costly. Probably not important in the scheme of things.